### PR TITLE
uploading passwordgen map and file

### PR DESCRIPTION
**Pull request by FPeterL**
Done with the basics code.

### CODE REVIEW

File: 7_week/passwordgen.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+def passwordgen(length):
+    pass_chars = string.ascii_letters + string.digits + "!@#$%^&*()"
+
+
+    password = ""
+    for i in range(length):
+        char = random.choice(pass_chars)
+        password = password + char
+
+    if length =< 8:
```
Comment (FeherPeter2) [inline 7_week/passwordgen.py]: syntax error, please use <=

File: 7_week/passwordgen.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        print("Figyelem: A jelszó túl rövid lehet.")
+
+    return pass_word
```
Comment (FeherPeter2) [inline 7_week/passwordgen.py]: this variable should be "password"

**Review by FeherPeter2 — CHANGES_REQUESTED** [review]
There are a few errors, please fix them

File: 7_week/passwordgen.py (hunk 1 above)
Comment (FPeterL) [reply 7_week/passwordgen.py]: syntax fixed to if length <= 8:

File: 7_week/passwordgen.py (hunk 2 above)
Comment (FPeterL) [reply 7_week/passwordgen.py]: variable fixed

File: 7_week/passwordgen.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import random
```
Comment (FeherPeter2) [inline 7_week/passwordgen.py:3]: Please add some description about the code

File: 7_week/passwordgen.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import string
+import os
```
Comment (FeherPeter2) [inline 7_week/passwordgen.py]: i dont think we need this import

File: 7_week/passwordgen.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+
+def passwordgen(length):
+    pass_chars = string.ascii_letters + string.digits + "!@#$%^&*()"
+
```
Comment (FeherPeter2) [inline 7_week/passwordgen.py:12]: should write some documentation about the funkcion and the variables

File: 7_week/passwordgen.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+    password = ""
+    for i in range(length):
+        char = random.choice(pass_chars)
+        password = password + char
+
+    if length <= 8:
+        print("Figyelem: A jelszó túl rövid lehet.")
```
Comment (FeherPeter2) [inline 7_week/passwordgen.py]: Should use enbglish instead of hungarian

File: 7_week/passwordgen.py [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+import string
+import os
+
+
+def passwordgen(length):
+    pass_chars = string.ascii_letters + string.digits + "!@#$%^&*()"
+
+
+    password = ""
+    for i in range(length):
```
Comment (FeherPeter2) [inline 7_week/passwordgen.py]: i variable isint in use

**Review by FeherPeter2 — CHANGES_REQUESTED** [review]
The code like fine but needs some refining, check the comments

File: 7_week/passwordgen.py (hunk 3 above)
Comment (FeherPeter2) [reply 7_week/passwordgen.py:3]: Description added

File: 7_week/passwordgen.py (hunk 4 above)
Comment (FeherPeter2) [reply 7_week/passwordgen.py]: os import removed

File: 7_week/passwordgen.py (hunk 6 above)
Comment (FeherPeter2) [reply 7_week/passwordgen.py]: language changed

File: 7_week/passwordgen.py (hunk 7 above)
Comment (FeherPeter2) [reply 7_week/passwordgen.py]: i changet to _

File: 7_week/passwordgen.py (hunk 5 above)
Comment (FeherPeter2) [reply 7_week/passwordgen.py:12]: documentation added

**Review by FeherPeter2 — APPROVED** [review]
Looks great, thanks for the nice work!

**Pull request merged**